iron-form.html

```diff
@@ -76,6 +76,16 @@
       withCredentials: {
         type: Boolean,
         value: false
+      },
+
+      /**
+      * Specify extra headers to be sent with the ajax request
```

Comment:
Looking at the property in `iron-ajax`, it also has the comment "Note: setting a `Content-Type` header here will override the value specified by the `contentType` property of this element.", which I think we should also add here.